build.sbt

```diff
@@ -2,7 +2,8 @@ import Dependencies._
 import sbt.url
 import sbtrelease.ReleaseStateTransformations.{checkSnapshotDependencies, commitNextVersion, commitReleaseVersion, inquireVersions, pushChanges, runClean, runTest, setNextVersion, setReleaseVersion, tagRelease}
 
-lazy val supportedScalaVersions = List("2.13.0", "2.12.8")
+scalaVersion := "2.13.0"
+lazy val supportedScalaVersions = List("2.13.0")
 ThisBuild / version := (version in ThisBuild).value
```

Comment:
I don't think you need the supportedScalaVersions unless you're trying to support more than one.

Reply:
I've removed supportedScalaVersions. 👍